configure EIP port and healthcheck ports from existing service with env var override (#173)
equinix/cloud-provider-equinix-metal: README.md

1. Create an Elastic IP, using the Equinix Metal API, Web UI or CLI
1. Put an arbitrary but unique tag on the EIP
1. When starting the CCM
   * set the [configuration][Configuration] for the control plane EIP tag, e.g. env var `METAL_EIP_TAG=<tag>`, where `<tag>` is whatever tag you set on the EIP

>    * (optional) set the port that the EIP should listen o; by default, it will be the same port as the `kube-apiserver` on the control plane nodes, but you can change it with `METAL_API_SERVER_PORT=<port>`

Reviewer: ```suggestion
   * (optional) set the port that the EIP should listen on; by default, or when set to `0`, it will use the same port as the `kube-apiserver` on the control plane nodes. This port can also be specified with `METAL_API_SERVER_PORT=<port>`
```
